### DISCUSSION TITLE
Getting information about contribution

### DISCUSSION THREAD

**Discussion by lagosr-github** [lichess-org/fishnet#225] [General]
I've been running fishnet at home in my 2 node kubernetes cluster for about a year.  and was wondering if there is a way to get an idea of what sort of impact that has on lichess.

From my own logs i know that i've processed 2.8M games -- just curious if there are stats on # of games that go to analysis per day on average.

![fishnet - Elastic (1)](https://user-images.githubusercontent.com/3108304/161002975-2538ccc3-87ca-4322-b572-8de411bad96a.png)

**Comment by niklasf** [comment]
Hi, thank you very much for running fishnet, especially so consistently!

About 2 billion games ([rated](https://database.lichess.org/) and unrated) have been played in the last year, and analysis is requested for about 5%. So roughly 2,8% of all fishnet analysis comes from your cluster.

The number of active keys fluctuates between 120 and 150, so this is one of the larger individual portions.

![image](https://user-images.githubusercontent.com/402777/161011621-ad52beed-cfaa-4ed1-b8dd-3ab3bc50d22a.png)

**Reply by yozurukt** [reply]
I just joined fishnet a few days ago and found this discussion while looking for the same information.
I'm currently processing around 20k games/day,  I also put together a simple dashboard for my stats.
I was wondering if there are any more recent numbers for roughly how many games fishnet processes per day nowadays?
<img width="1680" height="949" alt="image" src="https://github.com/user-attachments/assets/690cd63e-5a1e-48fd-8a92-506a108a1f84" />

**Reply by ounben** [reply]
Just a reminder why there is no Leaderboard:  [fishnet readme](https://github.com/lichess-org/fishnet/tree/9056bb9d772bc68bed7e23111448a3c99ff16dc0#is-there-a-leaderboard-of-contributors) , https://github.com/lichess-org/fishnet/discussions/209#discussioncomment-1493571 and https://github.com/lichess-org/fishnet/discussions/209#discussioncomment-1717572

In 2026.05 my User Database says this for your asked number:

- 311’928 analysis per day.
- 10 percent of all games in May had an analysis.

**Results DB** 
"has_analysis";"count"
"false";"81022641"
"true";"9669790"

**Lichess DB Total Games** 
https://database.lichess.org/
2026 - July | 29.1 GB | 89,288,421 
2026 - June | 28.2 GB | 86,483,328 
2026 - May | 29.7 GB | 90,887,615 


```
SELECT has_analysis, COUNT(*) 
FROM "lichess_db_pgn_headers_without_moves_2026_05"
GROUP BY has_analysis;
/* Betroffene Zeilen: 0  Gefundene Zeilen: 2  Warnungen: 0  Dauer von 1 Abfrage: 0.000 Sek. (+ 739.735 Sek. Netzwerk) */

```

**Comment by jim-perkins** [comment]
cool info.  I like the graphs.  looks like it took me nearly 2 years to run 837k jobs although I did take 3 months off

` journalctl -u fishnet2.service | head -n1;journalctl -u fishnet2.service|grep finished |wc -l
-- Logs begin at Wed 2020-04-22 18:48:55 CDT, end at Thu 2022-03-31 03:49:46 CDT. --
837503
`

**Comment by lagosr-github** [comment]
thanks for the info.  i play on lichess almost daily, and happy to be contributing to this service.

The cluster is dedicating 2 nodes to fishnet, using up to 12 cores total.   -- its my own home servers.  --  from a power consumption perspective, the house gets ~50% of its power from solar. -- so fairly clean.

jim - i run the logs through elastic-search and i threw together a dashboard from that data.  -- the cluster + services was my pandemic project at home during the first lockdown.
